src/modules/Basket/DeliveryForm.tsx

```diff
@@ -12,19 +13,159 @@ import {
   Stack,
   Text,
 } from '@chakra-ui/react'
-import { BasketTypes } from '../../types'
+import { BasketTypes, ReturnedOrder } from '../../types'
 import { ArrowBackIcon } from '@chakra-ui/icons'
 import InfoToPay from './InfoToPay'
+import {
+  useBasketContext,
+  useBasketDispatchContext,
+} from 'contexts/BasketContext'
+import { handleClick, makeOrder } from './makeOrderFuncs'
+import { postVoucher } from 'api'
+
+const STRIPE_SK = import.meta.env.VITE_STRIPE_SECRET_KEY
+const BASE_URL = import.meta.env.VITE_APP_BASE_URL
 
 interface Props {
   setSelectedBasketType: React.Dispatch<React.SetStateAction<BasketTypes>>
+  setOrderId: React.Dispatch<React.SetStateAction<number>>
 }
 
-const DeliveryForm = ({ setSelectedBasketType }: Props) => {
-  const [name, setName] = useState('')
-  const [phoneNumber, setPhoneNumber] = useState('')
-  const [deliveryType, setDeliveryType] = useState('self')
-  const [street, setStreet] = useState('')
+const getFromLocaleStorage = (key: string, defaultValue: string): string => {
+  const storedValue = localStorage.getItem(key)
+  return storedValue ? JSON.parse(storedValue) : defaultValue
```

Comment:
tip: for better readability ternar should be used in jsx, for describe functional logic use if/else